test/scripts/performance/mt02_verify_tenants_creation_performance.bash

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
```

Comment:
Minor change - can we rename the `.bash` file ending to `.sh` to match the rest of the scripts in the repo.

Reply:
Done. Thank you for comment

Reply:
@adam-cattermole , thank you for review. Regarding ` one tenant was stuck`, - I also saw it and wrote in report that attached to Jira. Maybe you want to add the same note to Jira. Thanks